app/src/main/res/values/strings.xml

```diff
@@ -124,6 +124,10 @@
     <string name="auto_correction_threshold_mode_aggressive">Aggressive</string>
     <!-- Option to suggest auto correction suggestions very aggressively. Auto-corrects to a word which has even large edit distance from typed word. [CHAR LIMIT=20] -->
     <string name="auto_correction_threshold_mode_very_aggressive">Very aggressive</string>
+    <!-- Option to show the text to be committed as the middle middle suggestion -->
+    <string name="center_suggestion_text_to_commit">Center text to commit</string>
+    <!-- Description for the center text to commit setting -->
+    <string name="center_suggestion_text_to_commit_summary">If enabled, the middle suggestion will show the text to be committed</string>
```

Comment:
I think it would be good to avoid to avoid "commit" in the name and description (no one used it in the related issue).
Also it's somewhat confusing, as the context of where the text is centered is not clear.

I'm not good at finding short and clear titles and descriptions, not sure how much I can help here...

Reply:
how about "Input preview suggestion"?
and description: "If enabled, the middle suggestion will display the text to be entered"

Reply:
I asked in the issue, maybe someone has a good idea. If not we can still change it later (though it would mean unnecessary work for translators)

Reply:
Maybe "middle suggestion always used"? Then for the description something like "If enabled, the middle suggestion will display the word selected for completion, whether the typed text was correct or not"? Add a bit more detail for what its doing. Obviously just some ideas

Reply:
"Always use middle suggestion" sounds good to me, at least I think this is hardest do mis-interpret.
And as summary something like "On pressing space or punctuation, the middle suggestion will be used". Or maybe "typed" or "entered"?

Reply:
Used or entered seem better, given no typing is taking place. Entered leaves less ambiguity, but used might be more clear. Your summary is much clearer than mine in fewer words lol